src/general/WorldGenerationSettings.cs

```diff
@@ -1,6 +1,158 @@
-﻿/// <summary>
+﻿using System;
+
+/// <summary>
 ///   Player configurable options for creating the game world
 /// </summary>
 public class WorldGenerationSettings
 {
+    public enum LifeOrigin
+    {
+        Vent,
+        Pond,
+        Panspermia,
+    }
+
+    public enum PatchMapType
+    {
+        Procedural,
+        Classic,
+    }
+
+    /*
+    Values for this particular object
+    */
+
+    public bool Lawk { get; set; }
+    public DifficultyPreset Difficulty { get; set; } = DifficultyPreset.Normal;
+    public LifeOrigin Origin { get; set; } = LifeOrigin.Vent;
+    public int Seed { get; set; } = new Random().Next();
+    public float MPMultiplier { get; set; } = 1;
+    public float AIMutationMultiplier { get; set; } = 1;
+    public float CompoundDensity { get; set; } = 1;
+    public float PlayerDeathPopulationPenalty { get; set; } = 1;
+    public float GlucoseDecay { get; set; } = 0.8f;
+    public float OsmoregulationMultiplier { get; set; } = 1;
+    public bool FreeGlucoseCloud { get; set; }
+    public PatchMapType MapType { get; set; } = PatchMapType.Procedural;
+    public bool IncludeMulticellular { get; set; } = true;
+    public bool EasterEggs { get; set; } = true;
+
+    /*
+    Static values for each difficulty preset
+    */
+
+    public static float GetMPMultiplier(DifficultyPreset preset)
```

Comment:
I think it would be a cleaner design if there was a Difficulty class which already had all of the right properties initialized for the difficulty level, that way these constant lookups would not be needed, and those difficulty presets could be loaded from a JSON file for easy tweaking by @buckly90 for balancing.